Sources/SwerverCore/Request/HttpRequest.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+public class HttpRequest {
+    
+    private let method: String
+    private let url: String
```

Comment:
Doesn't Swift have a specific URL object that can be used for your url?

Reply:
the http request class is called by the parser class to build an http object. I used string because the parser is parsing from a string request and since this story does not concern with the actual url itself, I left it as a string.